x-pack/solutions/observability/plugins/slo/server/lib/rules/slo_burn_rate/register.ts

```diff
@@ -93,7 +93,24 @@ export function sloBurnRateRuleType(
     },
     alerts: {
       context: SLO_RULE_REGISTRATION_CONTEXT,
-      mappings: { fieldMap: { ...legacyExperimentalFieldMap, ...sloRuleFieldMap } },
+      mappings: {
+        fieldMap: {
+          ...legacyExperimentalFieldMap,
+          ...sloRuleFieldMap,
+          dynamicTemplates: [
+            {
+              strings_as_keywords: {
+                path_match: 'kibana.alert.grouping.*',
```

Comment:
@benakansara I took the code snippet for the dynamic template from the meta ticket. Wondering if it is OK to use the ALERT_GROUPING variable here `path_match: `${ALERT_GROUPING}.*`?